googlesearch/user_agents.py

```diff
@@ -1,16 +1,213 @@
 import random
+import secrets
 
+def _get_useragent():
+    """
+    Returns a random user agent
 
-def get_useragent():
-    return random.choice(_useragent_list)
+    Returns:
+        str: Random user agent
 
-
-_useragent_list = [
+    """
+    return random.choice([
     'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:66.0) Gecko/20100101 Firefox/66.0',
```

Comment:
We only kept the common user agents since having a bunch was causing some issues.